lib/Fmt_ast.ml

```diff
@@ -594,7 +594,39 @@ and fmt_attributes = fmt_attributes_aux ~key:Attr.Key.Regular
 
 and fmt_item_attributes = fmt_attributes_aux ~key:Attr.Key.Item
 
-and fmt_floating_attributes = fmt_attributes_aux ~key:Attr.Key.Floating
+and fmt_attributes_and_docstrings_aux c ~key attrs =
+  let standalone, pro, space =
+    match key with
+    | Attr.Key.Regular | Attr.Key.Item ->
+        (false, break c.conf.doc_comments_padding 0, break 1 0)
```

Comment:
`doc_comments_padding` is now applied to more docs, are we sure it is not done in new places ? I couldn't find any.

Reply:
I think I already checked it